news/azure__use_short_config_name_at_artifact_upload

```diff
@@ -0,0 +1,23 @@
+**Added:**
```

Comment:
This file should have a `.rst` extension

Reply:
Thanks, btw here a build using modified templates: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=312124&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=84bc1e81-0e01-5010-923c-2b33ab27b5a4&l=9